aggregates/src/androidMain/kotlin/org/dhis2/mobile/aggregates/data/DataSetInstanceRepositoryImpl.kt

```diff
@@ -332,7 +335,22 @@ internal class DataSetInstanceRepositoryImpl(
             .uid(sectionUid)
             .blockingGet()
             ?.greyedFields()
-            ?.mapNotNull { it.dataElement()?.uid() }
+            ?.mapNotNull {
```

Comment:
can you think in a way to cover this fix by unit test to assert that specific fields are not editable?